src/components/PhotoList.jsx

```diff
@@ -57,4 +57,24 @@ export class PhotoList extends Component {
   }
 }
 
-export default Dimensions()(PhotoList)
+export class PhotoLists extends Component {
+  render () {
+    const { photoLists, containerWidth, selected, onPhotoToggle } = this.props
+    return (
+      <div>
+        { photoLists.map(photoList => {
+          return (<PhotoList
```

Comment:
@enguerran What do you think about the common role of `PhotoLists` and `Photoboard` ? I feel that `PhotoLists` just adds a `div` level into `PhotoBoard`.